### ISSUE TITLE
ignore node_modules

### ISSUE THREAD

**Issue by danielrohers** [mamute/node-express-tests#1]
this project is very useful

https://github.com/github/gitignore

**Comment by mamute** [comment]
Yeah, i'm using it now... The problem was I had a node_modules committed before the gitignore. :)

**Comment by danielrohers** [comment]
Ooow... classic

**Comment by mamute** [comment]
I wouldn't put this on github. Last minute decision... This was an collateral effect lol
